app/forms/state_file/md_eligibility_filing_status_form.rb

```diff
@@ -4,8 +4,8 @@ class MdEligibilityFilingStatusForm < QuestionsForm
 
     validates :eligibility_filing_status_mfj, presence: true
     validates :eligibility_homebuyer_withdrawal_mfj, presence: true, if: -> { eligibility_filing_status_mfj == "yes" }
-    validates :eligibility_homebuyer_withdrawal, presence: true, unless: -> { eligibility_filing_status_mfj == "yes" }
     validates :eligibility_home_different_areas, presence: true, if: -> { eligibility_filing_status_mfj == "yes" }
+    validates :eligibility_homebuyer_withdrawal, presence: true, unless: -> { eligibility_filing_status_mfj == "yes" }
```

Comment:
moved b/c that `if/unless` code was hard to detect difference & want to group things together